modules/builds-disabling-build-strategy-globally.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * assembly/builds
+// * builds/securing-builds-by-strategy.adoc
+
+
+[id='builds-disabling-build-strategy-globally-{context}']
+= Disabling access to a build strategy globally
+
+To prevent access to a particular build strategy globally, log in as a user with
+*cluster-admin* privileges, remove the corresponding role from the
+*system:authenticated* group, and apply the annotation
+`openshift.io/reconcile-protect: "true"` to protect them from changes between
```

Comment:
double check this with @openshift/sig-auth as think the name of the annotation may have changed.

Reply:
@enj would you mind confirming the name of this annotation, please? Thanks!

Reply:
@ericavonb, will one of you PTAL at this section and let us know who you think should confirm it for 4.0/4.1?

Reply:
It changed when we switched to the upstream RBAC group: use `rbac.authorization.kubernetes.io/autoupdate: "false"` instead